scripts/output/comparison/compareScenarios2.R

```diff
@@ -0,0 +1,140 @@
+  listofruns <- list(list(
+    period = "both",
+    set = format(Sys.time(), "%Y-%m-%d_%H.%M.%S"),
+    dirs = outputdirs))
+} else {
+  # This is the case if this script was called directly via Rscript
+  listofruns <- list(
+    list(
+      period = "both",
+      set = "cpl-Base",
+      dirs = c("C_SDP-Base-rem-5","C_SSP1-Base-rem-5","C_SSP2-Base-rem-5","C_SSP5-Base-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-PkBudg900",
+      dirs = c("C_SDP-PkBudg900-rem-5","C_SSP1-PkBudg900-rem-5","C_SSP2-PkBudg900-rem-5","C_SSP5-PkBudg900-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-PkBudg1100",
+      dirs = c("C_SDP-PkBudg1100-rem-5","C_SSP1-PkBudg1100-rem-5","C_SSP2-PkBudg1100-rem-5","C_SSP5-PkBudg1100-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-PkBudg1300",
+      dirs = c("C_SDP-PkBudg1300-rem-5","C_SSP1-PkBudg1300-rem-5","C_SSP2-PkBudg1300-rem-5","C_SSP5-PkBudg1300-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-NPi",
+      dirs = c("C_SDP-NPi-rem-5","C_SSP1-NPi-rem-5","C_SSP2-NPi-rem-5","C_SSP5-NPi-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-SDP",
+      dirs = c("C_SDP-Base-rem-5","C_SDP-NPi-rem-5","C_SDP-PkBudg1300-rem-5","C_SDP-PkBudg1100-rem-5","C_SDP-PkBudg1000-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-SSP1",
+      dirs = c("C_SSP1-Base-rem-5","C_SSP1-NPi-rem-5","C_SSP1-PkBudg1300-rem-5","C_SSP1-PkBudg1100-rem-5","C_SSP1-PkBudg900-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-SSP2",
+      dirs = c("C_SSP2-Base-rem-5","C_SSP2-NPi-rem-5","C_SSP2-PkBudg1300-rem-5","C_SSP2-PkBudg1100-rem-5","C_SSP2-PkBudg900-rem-5","C_SSP2-NDC-rem-5")),
+    list(
+      period = "both",
+      set = "cpl-SSP5",
+      dirs = c("C_SSP5-Base-rem-5","C_SSP5-NPi-rem-5","C_SSP5-PkBudg1300-rem-5","C_SSP5-PkBudg1100-rem-5","C_SSP5-PkBudg900-rem-5")))
+}
+
+# remove the NULL element
+listofruns <- listofruns[!sapply(listofruns, is.null)]
+
+# if no path in "dirs" starts with "output/" insert it at the beginning
+# this is the case if listofruns was created in the lower case above !exists("outputdirs"), i.e. if this script was not called via Rscript output.R
+for (i in 1:length(listofruns)) {
+  if (!any(grepl("output/", listofruns[[i]]$dirs))) {
+    listofruns[[i]]$dirs <- paste0("output/", listofruns[[i]]$dirs)
+  }
+}
+
+# ---- Start compareScenarios either on the cluster or locally ----
+
+start_comp <- function(outputdirs,
+                       shortTerm,
+                       outfilename,
+                       regionList,
+                       mainReg) {
+  if (!exists("slurmConfig")) {
+    slurmConfig <- "--qos=standby"
+  }
+  jobname <- paste0(
+      "cs2",
+      ifelse(outfilename == "", "", "-"),
+      outfilename,
+      ifelse(shortTerm, "-shortTerm", "")
+    )
+  cat("Starting ", jobname, "\n")
+  on_cluster <- file.exists("/p/projects/")
+  clcom <- paste0(
+    "sbatch --qos=standby",
```

Comment:
should be `paste0("sbatch", slurmConfig, …`

Reply:
Thanks for spotting that! Should this also be changed in compareScenarios1: https://github.com/remindmodel/remind/blob/bc8acd242c9958d779ef1b0e88520d00384d6f6b/scripts/output/comparison/compareScenarios.R#L56

Reply:
yes, please! I like the way you formulated it, avoiding redundancies and these errors in the future!